docs/user-guide/vfile-plugin.md

```diff
@@ -124,3 +124,11 @@ docker service create --replicas 1 -p 8080:80 --name web nginx
 ```
 Then on each host, make sure there is valid return of `curl 127.0.0.1:8080`.
 `Connection refused` error means the routing mesh of this Docker Swarm cluster is broken and needs to be fixed.
+
+### I got " VolumeDriver.Mount: Failed to blocking wait for Mounted state. Error: Timeout reached; BlockingWait is not complete.." when mounting a volume.
+We see this issue only on platforms which the availble disk space are low. On those platforms, docker swarm service takes longer time to up and running. We recommend to run this driver on platforms with enough available disk space. You can also try to increase the service start timeout value by specifing ```VFILE_TIMEOUT_IN_SECOND``` value when installing the plugin:
```

Comment:
Suggestion: 

When  available disk space are low, Docker Swarm service may take longer start a service. Generally it's better free up some e disk space. You can also try to increase the service start timeout value, controlled by ```VFILE_TIMEOUT_IN_SECOND``` env variable: